pkg/squashfsparser/squashfsparser.go

```diff
@@ -37,7 +37,7 @@ const (
 
 var (
 	// drwxr-xr-x administrator/administrator 66 2019-04-08 18:49 squashfs-root
-	fileLineRegex = regexp.MustCompile(`^([A-Za-z-]+)\s+([-\w]+|\d+)/([-\w]+|\d+)\s+(\d+)\s+(\d+-\d+-\d+)\s+(\d+:\d+)\s+(.*)$`)
+	fileLineRegex = regexp.MustCompile(`^([A-Za-z-]+)\s+([_\-\.\w]+|\d+)/([_\-\.\w]+|\d+)\s+(\d+)\s+(\d+-\d+-\d+)\s+(\d+:\d+)\s+(.*)$`)
```

Comment:
\w includes _ so I think you can leave that out.

Reply:
You also don't need to escape `.` and `-` when they're in `[]`